src/app/app-routing.module.ts

```diff
@@ -1,26 +1,23 @@
 import { NgModule } from '@angular/core';
 import { Routes, RouterModule, PreloadAllModules } from '@angular/router';
 import { DocumentSearchModule } from './modules/document-search/document-search.module';
+import { addInitializeModuleGuard } from '@onecx/angular-integration-interface';
+import { startsWith } from '@onecx/angular-webcomponents';
 
 export const routes: Routes = [
   {
-    path: '',
-    redirectTo: 'search',
-    pathMatch: 'full',
-  },
-  {
-    path: 'search',
+    matcher: startsWith('search'),
```

Comment:
With this change you are removing possibility to access DocumentSearchModule via empty path. You should do the following:
```typescript
{
    matcher: startsWith(''),
    redirectTo: 'search',
    pathMatch: 'full',
  },
```